invisible_cities/io/mcinfo_io.py

```diff
@@ -168,6 +172,37 @@ def __call__(self, mctables: (tb.Table, tb.Table, tb.Table, tb.Table),
         self.generator_table.flush()
 
 
+def copy_mc_info(files_in, h5out, event_list):
+    """
+    Copies MC info for selected events to an output file.
+    files_in :   List[str]
+                 List with the name of the input files.
+    file_out   : str
+                 The name of the output file.
+    event_list : List[int]
+                 List of events (event_id) to be copied.
+    """
+
+    writer = mc_info_writer(h5out)
+
+    for f in files_in:
+        try:
+            with tb.open_file(f, "r") as h5in:
+                run_number = get_run_number(h5in)
+                mc_info    = get_mc_info_safe(h5in, run_number)
```

Comment:
Related to what I said in another comment. Do not let it figure it out. This is a base function and I think it is safer and better to make it work only for the case it is meant to be used. In my opinion, it should complain if there is not a MC table in the input file.

Reply:
Apparently, this comment was confusing. I meant that `copy_mc_info` should not assume anything about whether the sign of run number means something or not. I would like `copy_mc_info` to assume whatever files come in to be MC files and have the relevant MC tables. If something goes wrong it should crash (there should be an explicit check to make the crash more obvious). Let the user of this function (aka the cities or the components) handle that situation (that is, calling `copy_mc_info` only for MC files).

Reply:
Done.